assets/models/system/mask_rcnn_swin-t-p4-w7_fpn_1x_coco/spec.yaml

```diff
@@ -21,12 +26,96 @@ properties:
     Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4,
     Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4,
     Standard_ND40rs_v2
-  model_id: mask_rcnn_swin-t-p4-w7_fpn_1x_coco
 tags:
   Preview: ''
+  SharedComputeCapacityEnabled: ''
```

Comment:
What's the significance of this tag?

Reply:
Unclear, as per ayush - if workspace is out of quota then still it deployed on shared compute. Mandetory field to pass validation.